Parse inst containing multiple colons

awailly opened the pull request:
- The current context_code will silently fail if the inst contains
  multiple colons. This is happening with C++ programs having std:: in
  the symbol name.

longld commented:
This will break x86 check, e.g: `jmp    DWORD PTR ds:0x804a00c` . Can you provide sample asm for C++?

awailly commented:
Sure, I created the following gist to underline the buggy one : 
https://gist.github.com/awailly/a3bafc916c39464f62b9

Maybe using `opcode = inst.split(": ")[-1].split()[0]` (with a space) could be enough ?
